SignalMessaging/environment/Environment.m

```diff
@@ -43,6 +43,11 @@ + (void)setCurrent:(Environment *)environment
     sharedEnvironment = environment;
 }
 
++ (void)clearCurrentForTests
+{
+    sharedEnvironment = nil;
```

Comment:
Alternately we could remove the new assert in the setter.

Reply:
I prefer it as written for now. 👍